File: C++/Assignments/fibonacci.cpp
#include <iostream>
using namespace std;

int main()
{

int inp_int, count =0, n1=1,n2 = 1;

cin >> inp_int;
for (int x =0; x < inp_int; x++)
{
  cout << n1<<endl;
  int nth = n1+ n2;
  n1 =n2;
  n2 = nth;

}



return 0;
}

File: Python/Assignments/fibonacci.py
inp_int = int(input("Please enter a positive integer greater 1: "))

count = 0

n1 =1
n2 = 1

for x in range(0, inp_int ):
  print(n1)
  nth = n1 +n2
  n1 =n2
  n2 = nth
  
